src/pages/home.tsx

```diff
@@ -1,23 +1,79 @@
 import { LinkButton } from "~components/Button";
 import { useRecentEvents } from "~utils/hooks/history";
+import { useEffect, useState } from "react";
+import Markdown from "react-markdown";
+import pjson from "../../package.json";
+import "./markdown.css";
 
 export const HomePage: React.FC = () => {
   const { data: recentEvents } = useRecentEvents(5);
 
+  const [markdownContent, setMarkdownContent] = useState<string>("");
+
+  useEffect(() => {
+    // Fetch the update notes as a markdown file
+    const fetchMarkdownContent = async () => {
+      try {
+        const response = await fetch("updateNotes.md");
+        const content = await response.text();
+        setMarkdownContent(content);
+      } catch (error) {
+        console.error("Error fetching Markdown content:", error);
+      }
+    };
+
+    fetchMarkdownContent();
+
+    const latestVersion = pjson.version;
+    const userVersion = localStorage.getItem("version");
+
+    if (userVersion && userVersion !== latestVersion) {
+      document.getElementById("markdown")?.classList.remove("hidden");
+    }
+    localStorage.setItem("version", latestVersion);
+  }, []); // Run this effect only once, when the component mounts
+
   return (
-    <section className="mt-4 max-w-full">
-      {recentEvents?.map((event) => (
-        <LinkButton
-          to={`/${event.sku}`}
-          className="w-full max-w-full mt-4"
-          key={event.sku}
+    <main>
+      <aside className="text-right">
+        <button
+          className="bg-cyan-900 text-right ml-auto mt-2 py-1 px-3 rounded-md"
+          onClick={() =>
+            document.getElementById("markdown")?.classList.remove("hidden")
+          }
+        >
+          View Update Notes
+        </button>
+      </aside>
+
+      <section className="max-w-full">
+        {recentEvents?.map((event) => (
+          <LinkButton
+            to={`/${event.sku}`}
+            className="w-full max-w-full mt-4"
+            key={event.sku}
+          >
+            <p className="text-sm">
+              <span className=" text-emerald-400 font-mono">{event.sku}</span>
+            </p>
+            <p className="">{event.name}</p>
+          </LinkButton>
+        ))}
+      </section>
+      <div
```

Comment:
You should use the [`<Dialog>`](https://github.com/brenapp/referee.fyi/blob/main/src/components/Dialog.tsx) component here, as it's better for accessibility (and does a lot of the styling for you).

Additionally, it will have an `open` prop you can pass, which I think is a bit cleaner the adding a removing the class via classList.

See the rules dialog for a good example that's pretty similar to what you're doing